apps/test/unit/turtle/turtleTest.js

```diff
@@ -175,6 +175,33 @@ describe('Artist', () => {
     });
   });
 
+  describe('pointTo', () => {
+    it('can point to 45 degrees', () => {
+      let angle = 45;
+      let blockId = "block_id_4";
+      let artist = new Artist();
+
+      artist.visualization = new Artist.Visualization();
+      const pointToSpy = sinon.spy(artist.visualization, 'pointTo');
```

Comment:
Could spy `setHeading` as well for completeness, but this is fine as-is.

Reply:
Test added to stub setHeading.